src/commands/function-flush.json

```diff
@@ -0,0 +1,37 @@
+{
+    "FLUSH": {
+        "summary": "PATCH__TBD__29__",
+        "complexity": "PATCH__TBD__28__",
```

Comment:
where did you copy these from?
need to fill them.

Reply:
Its all over the functions commands, I will complete them all in a separate PR?

Reply:
ok, you can do that in a separate PR (just don't forget)..
since this one is probably gonna be the first one to be merged, i don't mind you do that here either.

Reply:
#9953